Mathlib/MeasureTheory/Group/MeasurableEquiv.lean

```diff
@@ -201,3 +203,29 @@ def divLeft [MeasurableMul G] [MeasurableInv G] (g : G) : G ≃ᵐ G where
   measurable_invFun := measurable_inv.mul_const g
 
 end MeasurableEquiv
+
+namespace MeasureTheory.Measure
+variable {G A : Type*} [Group G] [AddCommGroup A] [DistribMulAction G A] [MeasurableSpace A]
```

Comment:
Can this be made to work for `Monoid`s by using `μ.comap (DomMulAct.mk.symm g • ·)` instead?

Reply:
Without `Group G`, I can't even prove
```
lemma dmaSMul_apply (μ : Measure A) (g : Gᵈᵐᵃ) (s : Set A) :
    (g • μ) s = μ (DomMulAct.mk.symm g • s)
```
so I am tempted to say "No"

Reply:
Here are a few lemmas I've written to modify the instance to use `comap`: #20218